autogpt_platform/frontend/src/lib/autogpt-server-api/client.ts

```diff
@@ -600,6 +600,14 @@ export default class BackendAPI {
     await this._request("PUT", `/library/agents/${libraryAgentId}`, params);
   }
 
+  /**
+   * Clone Library Agent and its underyling graph&nodes.
+   * User must own
+   */
+  deepCloneLibraryAgent(libraryAgentId: LibraryAgentID): Promise<LibraryAgent> {
```

Comment:
nit: `deepClone` is a technical detail that doesn't matter to the client, right? Could be `clone` or `fork` instead.